### ISSUE TITLE
Example doesnt work

### ISSUE THREAD

**Issue by lunfardo** [microsoft/Win2D#899]
Getting this error

Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0503	A value of type 'CanvasControl' cannot be added to a collection or dictionary of type 'UIElementCollection'.	15

**Comment by getrou** [comment]
Could we get a bit more information? For example, are you referring to one of our examples in the example gallery, and is this using Win2D for UWP or Win2D for WinUI3?

**Comment by spncrgr** [comment]
I'm getting the same error. I'm following a [Learn module](https://learn.microsoft.com/en-us/training/modules/get-started-with-visual-studio-for-windows10-app-dev/3-nuget), but I also see the same example in the [Introduction section of the Documentation](https://microsoft.github.io/Win2D/WinUI2/html/Introduction.htm).

My issue may have stemmed from the fact that when I searched for `Win2D.uwp` in the NuGet Package Manager UI (VS2022: Project > Manage NuGet Packages), I didn't see any exact matches (at least not on the first few pages). The `Microsoft.Graphics.Win2D` package did show up in the results, so I installed it, thinking that maybe the `Win2D.uwp` package had been renamed, and the docs were out-of-date. However, when I continued the tutorial, I received the error reported by OP.

I've worked around it with the NuGet Package Manager Console instead of the UI. When I searched for `Win2D.uwp` in the console, it found a match. I removed the `Microsoft.Graphics.Win2D` package and installed the `Win2D.uwp` package from the command line, and the app started running.
